src/style/app.module.css

```diff
@@ -21,6 +21,15 @@
   --loader-color: #febc59;
 
   /* Miscellaneous */
+  --title-gray: #5e5e5e;
+  --bs-btn-bg: #212529;
+  --grey-color: #a5a5a5;
+  --light-green: #98e0b6;
```

Comment:
1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
    1. https://www.figma.com/design/dmKt00m9GVSeA1nebnkxql/Untitled?node-id=0-1&node-type=canvas&t=TPzfh1DE9IxjzsEe-0
3. There are appropriate blue/grey colors that have been used in other screens

![image](https://github.com/user-attachments/assets/d7e58984-3ab0-4e60-9512-d1328d05b20d)